hr_holidays_hour/models/hr_employee.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    @api.multi
+    def _compute_leaves_count(self):
```

Comment:
I don't think is out of scope. I could remove this method, no problem, but then the user will read the computed leaves in days instead of hours. At the end I will create for our customers and extra module to convert this amount to hours. So if for you is important to remove it, I will do it ... do you confirm?

Reply:
OK, I see, then it's not out of scope indeed. Leave it :smiley: